src/libaktualizr/package_manager/ostreemanager.cc

```diff
@@ -317,24 +318,24 @@ GObjectUniquePtr<OstreeDeployment> OstreeManager::getStagedDeployment() const {
 }
 
 GObjectUniquePtr<OstreeSysroot> OstreeManager::LoadSysroot(const boost::filesystem::path &path) {
-  OstreeSysroot *sysroot = nullptr;
+  OstreeSysroot *sysroot_raw = nullptr;
```

Comment:
I think you can use `.reset()` to update the value of a `GObjectUniquePtr` and forego the sysroot_raw intermediary which can potentially be misused.

Reply:
Good call. Done.